shims/common/src/main/scala/io/glutenproject/GlutenConfig.scala

```diff
@@ -709,19 +709,19 @@ object GlutenConfig {
       .internal()
       .doc(
         "By default, the supported codecs are lz4 and zstd. " +
-          "When spark.gluten.sql.columnar.shuffle.codecBackend=qat, the supported codec is gzip. " +
-          "When spark.gluten.sql.columnar.shuffle.codecBackend=iaa, the supported codec is gzip.")
+          "When spark.gluten.sql.columnar.shuffle.codecBackend=qat," +
+          "the supported codecs are gzip and zstd. " +
+          "When spark.gluten.sql.columnar.shuffle.codecBackend=iaa," +
+          "the supported codec is gzip.")
       .stringConf
-      .transform(_.toUpperCase(Locale.ROOT))
-      .checkValues(
-        GLUTEN_SHUFFLE_SUPPORTED_CODEC ++ GLUTEN_QAT_SUPPORTED_CODEC ++ GLUTEN_IAA_SUPPORTED_CODEC)
+      .transform(_.toLowerCase(Locale.ROOT))
       .createOptional
 
   val COLUMNAR_SHUFFLE_CODEC_BACKEND =
     buildConf(GlutenConfig.GLUTEN_SHUFFLE_CODEC_BACKEND)
       .internal()
       .stringConf
-      .transform(_.toUpperCase(Locale.ROOT))
+      .transform(_.toLowerCase(Locale.ROOT))
```

Comment:
and here

Reply:
Same as above

Reply:
Most of config options only support lower case, consistence is good. But if you aim to support both, I don't against it.

Reply:
Thanks. We need to support both for backward compatibility.